src/game/CPlayerManager.cpp

```diff
@@ -218,6 +232,37 @@ void CPlayerManagerImpl::HandleEvent(SDL_Event &event) {
     // Events coming in are for the next frame to be sent.
     // FrameFunction *ff = &frameFuncs[(FUNCTIONBUFFERS - 1) & (itsGame->frameNumber + 1)];
 
+    if (event.type == itsGame->itsApp->ControllerAxisEventType()) {
+        ControllerAxisState *state = (ControllerAxisState *)event.user.data1;
+        float mult = pow((abs(state->current) / 32768.0), controllerCurveExp);
```

Comment:
This is basically `power(abs(state->current * 2.0 / FIX(1)), controllerCurveExp)`.  It might be nice have a setting such as `controllerCurveMult` to change that 2.0 value?  So, `abs(state->current * controllerCurveMult / FIX(1), controllerCurveExp)`.

Reply:
on 2nd thought, maybe that should multiply outside of the pow() call...`controllerCurveMult * pow(abs(state->current * 2.0 / FIX(1)), controllerCurveExp)` ???

Reply:
I did experiment with multipliers in addition to the power curve, and may eventually go that way again. But there are some other things I want to experiment with first (adjusting x/y to fit a circle instead of scaled on an axis, or adjusting angles instead of "mouse" points). Also, 32768 wasn't chosen because of anything related to Avara's fixed point system, the SDL controller axis values are a signed 16-bit number, so that basically just scales them to 0-1 before applying the power.